package.json

```diff
@@ -64,14 +66,18 @@
     "debug": "^2.3.3",
     "electron-installer-dmg": "^0.1.2",
     "electron-packager": "^8.4.0",
+    "electron-sudo": "malept/electron-sudo#fix-linux-sudo-detection",
```

Comment:
Just FYI, this can be changed to a version once https://github.com/automation-stack/electron-sudo/pull/41 is merged & released.